.github/workflows/test_merge_requests.yml

```diff
@@ -61,4 +61,4 @@ jobs:
           NAME=mailserver-testing:ci
           bash -c 'make generate-accounts tests'
         env:
-          CI: true
+          CI: true
```

Comment:
Interesting change. On my phone, these lines look identical :D I guess a tab vs. spaces thing. Looks funny though:)

Reply:
Oh weird... I can bring in the same file from master and it doesn't show any changes to commit... Unsure how this happened 🤔

Reply:
> LGTM 👍🏼
> 
> Very nice additions!
> I think this PRs paves the way for bringing some of the logic of `setup.sh` in the container itself to make `setup.sh` independent of the mail server version again. I will have a look after this PR is merged :D

Thanks! This can be introduced in the next minor release. How do we handle preparing things for certain versions?